src/utils/binarySearch.ts

```diff
@@ -0,0 +1,15 @@
+export const binarySearchOfPouchDbDocs = (
```

Comment:
This seems to be unused

Reply:
`src/containers/CommentsContainer/commentsContainer.helper.tsx` and also in idea home page, detail page

Reply:
👍